dev-setup.md

```diff
@@ -0,0 +1,185 @@
+  - enterprise-2022.1
+  - enterprise-2022.2
+  - enterprise-2023.1
+  - enterprise-2023.2
+  - enterprise-2023.3
+  - enterprise-2024.1
+  - enterprise-2025.1
+  - enterprise-2026.1
+  - siren-tests
+
+EMAIL_SENDER: "qabot@scylladb.com"
+EMAIL_SENDER_USER: ""
+EMAIL_SENDER_PASS: ""
+EMAIL_SERVER: "smtp.gmail.com"
+EMAIL_SERVER_PORT: "587"
+
+JOB_VALIDITY_PERIOD_DAYS: 30
+```
+
+#### Github OAuth App setup (getting :
+1. GITHUB_CLIENT_ID and GITHUB_CLIENT_SECRET
+   1. go to your Account Settings (top right corner) -> Developer settings (left pane) -> OAuth Apps
+   2. Click Create New OAuth App button
+   3. Fill the fields (app name: `argus-dev`, homepage URL `http://localhost:5000`, Auth callback URL: `http://localhost:5000/profile/oauth/github`)
+   4. Confirm and get the tokens/ids required for config - visible on the page
+
+#### Create Jenkins token for your account
+1. Click your username in top right corner 
+2.  write down user id
+3. Go to `Configure` from left panel
+4. Click `Add new Token` and name it `argus-dev`
+5. Get it and paste to config to `JENKINS_API_TOKEN` param
+
+## Configure database
+
+### Start database using docker-compose
+```bash
+docker-compose -f dev-db/docker-compose.yaml up -d
+```
+
+### create keyspace
+```bash
+docker exec -it argus_alpha_1 cqlsh --user cassandra --password cassandra -e "CREATE KEYSPACE argus WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor' : 1};"
+```
+
+### create all tables
+```bash
+FLASK_DEBUG=1 FLASK_ENV=development FLASK_APP=argus_backend:argus_app poetry run flask cli sync-models
+```
+
+### import jenkins jobs
+```bash
+FLASK_DEBUG=1 FLASK_ENV=development FLASK_APP=argus_backend:argus_app poetry run flask cli scan-jenkins
+```
+
+### import data
+#### Get sample data
+1. From S3
+```bash
+aws s3 cp s3:....  # todo
+```
+2. From prod db
+   1. Configure prod db (setup argus.local.yaml) by creating a ssh tunnel:
+   ```bash
+   # get argus-web details from the tribe
+   ssh argus-web -L 127.0.0.10:9042:10.10.2.62:9042 -L 127.0.0.11:9042:10.10.2.157:9042 -L 127.0.0.12:9042:10.10.2.74:9042 -L 127.0.0.10:19042:10.10.2.62:19042 -L 127.0.0.11:19042:10.10.2.157:19042 -L 127.0.0.12:19042:10.10.2.74:19042
+   ```
+    argus.local.yaml
+   ```yaml
+      # SCYLLA_CONTACT_POINTS:
+      - 127.0.0.10
+      - 127.0.0.11
+      - 127.0.0.12
+      # get db details from the tribe
+   ```
+Modify `release` variable in `dev-db/export_data.py` to your need.
+Run multiple times to download all the releases you need.
+```bash
+python dev-db/export_data.py
+```
+
+#### Import it
+```bash
+python dev-db/import_data.py
+```
+Sporadical errors like `Test entity missing for key "enterprise-2023.1/upgrade-with-raft/rolling-upgrade-centos8-with-raft-test", run won't be visible until this is corrected`
+can be ignored.
+
+## Run the application
+Compile frontend files from `/frontend` into `/public/dist`. Add --watch to recompile files on change.
+```bash
+yarn webpack --watch
```

Comment:
Apologies, meant this

Reply:
like: `WEBPACK_ENVIRONMENT=development yarn webpack --watch`?

Reply:
Yes

Reply:
fixed. Also moved this dev-doc.md to docs folder